Source/WebInspectorUI/UserInterface/Views/CompletionSuggestionsView.js

```diff
@@ -175,10 +175,12 @@ WI.CompletionSuggestionsView = class CompletionSuggestionsView extends WI.Object
         if (typeof selectedIndex === "number")
             this._selectedIndex = selectedIndex;
 
+        let elementToSelect;
         for (let [index, completion] of completions.entries()) {
             var itemElement = document.createElement("div");
             itemElement.classList.add("item");
-            itemElement.classList.toggle("selected", index === this._selectedIndex);
+            if (index === this._selectedIndex)
+                elementToSelect = itemElement;
```

Comment:
rather than go through the trouble of this on every iteration, perhaps we drop `elementToSelect` entirely and just do something like this after the `for`?
```js
let selectedItemElement = this._selectedItemElement;
if (selectedItemElement) {
    selectedItemElement.classList.add("selected");
    selectedItemElement.scrollIntoViewIfNeeded(false);
}
```
(note that `_selectedItemElement` is actually a `get` method)